<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1650</width>
    <height>1102</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1650</width>
    <height>1100</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1650</width>
    <height>1200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QStackedWidget" name="stackedWidget_settings">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>10000</width>
          <height>10000</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>1</number>
        </property>
        <widget class="QWidget" name="page_1">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QWidget {
    background-color: black;
}
</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout" stretch="1,11">
          <property name="spacing">
           <number>0</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QPushButton" name="open_file">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>200</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>53</pointsize>
                <italic>true</italic>
                <bold>true</bold>
                <underline>false</underline>
                <strikeout>false</strikeout>
                <kerning>true</kerning>
               </font>
              </property>
              <property name="cursor">
               <cursorShape>ForbiddenCursor</cursorShape>
              </property>
              <property name="styleSheet">
               <string notr="true">QPushButton {
    background-color: #995200; /* Розовый цвет (подставьте нужный цвет) */
    border: 2px solid #995200; /* Цвет рамки */
    color: black; /* Цвет текста */
    padding: 5px 10px; /* Отступы вокруг текста */
    border-radius: 5px; /* Закругленные углы */
}

QPushButton:hover {
    background-color: #f80; /* Новый цвет при наведении (подставьте нужный цвет) */
}
</string>
              </property>
              <property name="text">
               <string>open file</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="label_gif">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>40</width>
              <height>40</height>
             </size>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="margin">
             <number>0</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="page">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <property name="spacing">
           <number>12</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>12</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>12</number>
          </property>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QPushButton" name="new_file">
              <property name="cursor">
               <cursorShape>OpenHandCursor</cursorShape>
              </property>
              <property name="text">
               <string>NEW FILE</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QPushButton" name="gif">
                <property name="cursor">
                 <cursorShape>OpenHandCursor</cursorShape>
                </property>
                <property name="text">
                 <string>GIF</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="render">
                <property name="cursor">
                 <cursorShape>OpenHandCursor</cursorShape>
                </property>
                <property name="text">
                 <string>RENDER</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="position">
              <property name="cursor">
               <cursorShape>OpenHandCursor</cursorShape>
              </property>
              <property name="text">
               <string>MOVE</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="settings">
              <property name="cursor">
               <cursorShape>OpenHandCursor</cursorShape>
              </property>
              <property name="text">
               <string>SETTINGS</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Expanding</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="info_path">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <kerning>true</kerning>
               </font>
              </property>
              <property name="lineWidth">
               <number>0</number>
              </property>
              <property name="midLineWidth">
               <number>0</number>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
              <property name="wordWrap">
               <bool>true</bool>
              </property>
              <property name="margin">
               <number>0</number>
              </property>
              <property name="indent">
               <number>-1</number>
              </property>
              <property name="openExternalLinks">
               <bool>false</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="info_vertex">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="info_edg">
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
